src/Microsoft.ML.Transforms/Microsoft.ML.Transforms.csproj

```diff
@@ -6,6 +6,10 @@
     <DefineConstants>CORECLR</DefineConstants>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <NoWarn>CS8002;1591</NoWarn>
```

Comment:
Interesting that it isn't strong named.  That will cause issues on desktop.  We could ask for it to be strong named, or build our own copy.

Also this can be
```
<NoWarn>$(NoWarn);CS8002</NoWarn>
```

And placed in the main PropertyGroup @ line 7.

Reply:
FYI this is fixed with my changes [https://github.com/zeahmed/machinelearning/pull/4](https://github.com/zeahmed/machinelearning/pull/4)

---
In reply to: [209299533](https://github.com/zeahmed/machinelearning/pull/3#discussion_r209299533) [](ancestors = 209299533)